Simplenote/SimplenoteAppDelegate.m

```diff
@@ -163,7 +165,7 @@ - (void)handleGetURLEvent:(NSAppleEventDescriptor *)event withReplyEvent:(NSAppl
 
 - (void)setupCrashLogging
 {
-    [CrashLogging startWithSimperium: self.simperium];
+    self.crashLogging = [[CrashLogging alloc] initWithSimperium:self.simperium];
```

Comment:
```suggestion
    self.crashLogging = [[CrashLogging alloc] initWithSimperium:self.simperium];
    [self.crashLogging start];
```

We need to call `start` on the `CrashLogging` object in order to initialize the SDK internally.

Reply:
🤦‍♂️

Reply:
If it's a common pattern it could have been nice to add a `static func start(with simperium: Simperium)` function in the lib that would be just a wrapper around `init(simperium:)` + `start()` just for nicer API… bwah, too late and not worth it now ^^